### PR TITLE
remove news summary dataset from repo, download using script in sample

### PR DESCRIPTION
**Pull request by ManojBableshwar**
# Description

# Checklist

- [ ] I have read the [contribution guidelines](https://github.com/Azure/azureml-examples/blob/main/CONTRIBUTING.md)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by adrosa — APPROVED** [review]

File: sdk/python/foundation-models/system/finetune/summarization/news-summary.ipynb [hunk 1]
```diff
@@ -504,10 +520,8 @@
     "import json\n",
     "\n",
     "test_df = pd.read_json(\n",
-    "    \"./news-summary-dataset/test.jsonl\", orient=\"records\", lines=True\n",
+    "    \"./news-summary-dataset/small_test.jsonl\", orient=\"records\", lines=True\n",
```
Comment (adrosa) [inline sdk/python/foundation-models/system/finetune/summarization/news-summary.ipynb:523]: this does not solve the copyright issue

**Review by adrosa — CHANGES_REQUESTED** [review]

File: sdk/python/foundation-models/system/finetune/summarization/news-summary.ipynb [hunk 2]
```diff
@@ -85,9 +85,9 @@
     "except:\n",
     "    workspace_ml_client = MLClient(\n",
     "        credential,\n",
-    "        subscription_id=\"<SUBSCRIPTION_ID>\",\n",
-    "        resource_group_name=\"<RESOURCE_GROUP>\",\n",
-    "        workspace_name=\"<WORKSPACE_NAME>\",\n",
+    "        subscription_id = \"<SUBSCRIPTION_ID>\"\n",
```
[the remark this answers is not in the document]
Comment (adrosa) [reply sdk/python/foundation-models/system/finetune/summarization/news-summary.ipynb]: done!

**Comment by adrosa** [comment]
to appease black
```
pip install "black[jupyter]"
black cli/foundation-models/system/finetune/summarization/download-dataset.py
black sdk/python/foundation-models/system/finetune/summarization/news-summary.ipynb
```

**Review by Man-MSFT — APPROVED** [review]

File: cli/foundation-models/system/finetune/summarization/download-dataset.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
+# import library to parse command line arguments
+import argparse, os
+import pandas as pd
+import os
```
Comment (Man-MSFT) [inline cli/foundation-models/system/finetune/summarization/download-dataset.py:4]: already imported on line 1

File: cli/foundation-models/system/finetune/summarization/news-summary.sh [hunk 4]
```diff
@@ -3,9 +3,9 @@ set -x
 # the data files are available in the same folder as the above notebook
 
 # script inputs
-subscription_id="<SUBSCRIPTION_ID>"
-resource_group_name="<RESOURCE_GROUP>"
-workspace_name="<WORKSPACE_NAME>"
+subscription_id="21d8f407-c4c4-452e-87a4-e609bfb86248" #"<SUBSCRIPTION_ID>"
```
Comment (Man-MSFT) [inline cli/foundation-models/system/finetune/summarization/news-summary.sh:6]: What ID is this?

Comment (Man-MSFT) [reply cli/foundation-models/system/finetune/summarization/news-summary.sh:6]: Looks like there are test values left here, we need to remove these

File: cli/foundation-models/system/finetune/summarization/download-dataset.py [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    lines=True,
+)
+test_df.sample(frac=0.2).to_json(
+    os.path.join(args.download_dir, "small_test.jsonl"), orient="records", lines=True
+)
+
+
+# generate sample scoring data
+# read ./news-summary-dataset/small_test.jsonl into a pandas dataframe
+import pandas as pd
```
Comment (Man-MSFT) [inline cli/foundation-models/system/finetune/summarization/download-dataset.py:71]: already imported way up top

**Review by adrosa — APPROVED** [review]

**Review by chrjia — APPROVED** [review]

**Comment by ManojBableshwar** [comment]
will implement changes suggested by Man in a new PR as i need to get this in quick to unblock //build deliverables

**Review by adrosa — APPROVED** [review]

**Review by akshaya-a — APPROVED** [review]
follow up to address comments

**Pull request merged**